cmd/commands/cmd_invoice.go

```diff
@@ -116,6 +117,12 @@ var AddInvoiceCommand = cli.Command{
 				"use on a blinded path. The flag may be " +
 				"specified multiple times.",
 		},
+		cli.StringFlag{
+			Name: "blinded_path_incoming_channel_list",
+			Usage: "The chained channels specified via channel " +
+				"id (separated by commas), starting from a " +
+				"channel which points to the receiver node.",
```

Comment:
Micro nit: I think the receiver here can only be our node in this context. We cannot build a blinded route for now for another receiver, otherwise we would also need to supply the encryption keys for the blinded path. So I think we can just say self node ? wdyt